src/Drawing/BasicDrawing.cpp

```diff
@@ -34,10 +34,11 @@ void BasicDrawing::DrawElipse(const LVector2& center, const LColor& color, const
     ::DrawEllipse(center.X, center.Y, radius.X, radius.Y, c);
 }
 
-void BasicDrawing::DrawRectangle(const LVector2 &center, const LColor &color, const LVector2 &halfSize) {
+void BasicDrawing::DrawRectangle(const LVector2 &center, const LColor &color, const LVector2 &halfSize, const float rotation) {
     if (!CanDraw()) return;
     RColor c = ConvertEngineColor(color);
-    ::DrawRectangle(center.X - halfSize.X, center.Y - halfSize.Y, halfSize.X * 2, halfSize.Y * 2, c);
+    ::DrawRectanglePro(::Rectangle({center.X - halfSize.X, center.Y - halfSize.Y, halfSize.X * 2, halfSize.Y * 2}), 
```

Comment:
Isn't Rectangle top-left based, so we need it to be `center.Y + halfSize.Y`?

Reply:
tis but this is like Godot (or how most render APIs do screen drawing) where top-left is (0, 0) and bottom right is max screen size

Reply:
Weird